tests/microos/disk_boot.pm

```diff
@@ -28,6 +28,24 @@ sub run {
         shift->wait_boot(bootloader_time => 300);
     }
     microos_login;
+    record_kernel_audit_messages();
+}
+
+
+=head2 record_kernel_audit_messages
+
+Record the SELinux messages before the auditd daemon has been started, if present. If there are no such entries, this function has no effect.
+
+=cut
+
+sub record_kernel_audit_messages {
+    my $output = script_output("journalctl -k | grep 'audit:.*avc:'");
+    return unless ($output);    # Don't log anything if there is no output
+    record_info("Audit", "Kernel audit messages:\n\n$output");
+    # Upload the same log and don't fail on errors (supplemental material)
+    script_run("journalctl -k | grep 'audit:.*avc:' > /var/tmp/k-audit.log");
```

Comment:
```suggestion
    script_run("journalctl -k -u audit -u avc > /var/tmp/k-audit.log");
```

Reply:
This does not work. `audit` is not a unit.

Reply:
OK, I don't have better idea then :smile:
![image](https://user-images.githubusercontent.com/1254493/207311117-a1221b80-662f-44a0-9a68-6ff93564a439.png)

Reply:
That's ok. Me neither.